test/database/sorted.js

```diff
@@ -334,7 +334,7 @@ describe('Sorted Set methods', () => {
 		});
 
 		it('should work with big arrays (length > 100) ', async function () {
-			this.timeout(50000);
+			this.timeout(100000);
```

Comment:
This is mildly concerning -- does it have significant overhead?

Reply:
This is unrelated to the PR actually, this test creates very big sorted sets and was failing randomly on psql